ui/package.json

```diff
@@ -89,7 +89,6 @@
     "eslint-plugin-vue": "^7.0.0",
     "less": "^3.0.4",
     "less-loader": "^5.0.0",
-    "node-sass": "^4.0.0",
```

Comment:
cc @shwstppr @davidjumani @utchoang - pl check if we need node-sass? Removing this unblocks us to use any node version (I've tested both node v16 and v18). I did some quick build and run checks and I don't see any issues.